x-pack/filebeat/module/activemq/audit/test/audit.log-expected.json

```diff
@@ -0,0 +1,50 @@
+[
+    {
+        "activemq.audit.action": "called org.apache.activemq.broker.jmx.QueueView.retryMessages[] at 27-11-2019 08:45:57,213",
+        "activemq.audit.thread": "qtp443290224-47",
+        "activemq.audit.user": "anonymous",
+        "event.dataset": "activemq.audit",
+        "event.module": "activemq",
+        "fileset.name": "audit",
+        "input.type": "log",
+        "log.level": "INFO",
+        "log.offset": 0,
+        "service.type": "activemq"
+    },
+    {
+        "activemq.audit.action": "called org.apache.activemq.broker.jmx.QueueView.retryMessages[] at 27-11-2019 08:45:57,229",
+        "activemq.audit.thread": "qtp443290224-45",
+        "activemq.audit.user": "admin",
+        "event.dataset": "activemq.audit",
+        "event.module": "activemq",
+        "fileset.name": "audit",
+        "input.type": "log",
+        "log.level": "INFO",
+        "log.offset": 127,
+        "service.type": "activemq"
+    },
+    {
+        "activemq.audit.action": "requested /admin/createDestination.action [JMSDestination='test' JMSDestinationType='queue' secret='4eb0bc3e-9d7a-4256-844c-24f40fda98f1' ] from 127.0.0.1",
+        "activemq.audit.thread": "qtp12205619-39",
+        "activemq.audit.user": "admin",
+        "event.dataset": "activemq.audit",
+        "event.module": "activemq",
+        "fileset.name": "audit",
+        "input.type": "log",
+        "log.level": "WARN",
+        "log.offset": 250,
+        "service.type": "activemq"
+    },
+    {
+        "activemq.audit.action": "requested /admin/purgeDestination.action [JMSDestination='test' JMSDestinationType='queue' secret='eff6a932-1b58-45da-a64a-1b30b246cfc9' ] from 127.0.0.1",
+        "activemq.audit.thread": "qtp12205619-36",
+        "activemq.audit.user": "guest",
```

Comment:
We could put these potentially common fields at the module level (as `activemq.action`, `activemq.user`, `activemq.thread`, `activemq.caller`...) so if several datasets generate events with these fields they can be more easily filtered and corelated.

Reply:
So far the only common field is `activemq.thread`. I put it in the common namespace.

Reply:
I would put all that make sense in any context for the ActiveMQ service. For example any time we create an event with an ActiveMQ user we should put it in a common field, like `activemq.user`.
If action is specific of the audit subsystem then it can be ok to leave it as `activemq.audit.action`, but I would put at the module leven anything that has always the same meaning in the service.

Reply:
Thanks for the comment, @jsoriano . Fixed!